homework/test/test-list.js

```diff
@@ -1,8 +1,13 @@
-describe('Test > ', function() {
-  beforeEach(function() {
-  });
+var assert = require('assert');
+var Test = function(){};
 
-  it('will test some pure functions', function() {
-    // Write any pure function assertion here.
-  });
-});
+Test.prototype.add = function (n1,n2){
```

Comment:
It's better to define it in one of the managers, rather than to write it in the test file.